neurom/io/hdf5.py

```diff
@@ -81,7 +81,7 @@ class H5(object):
     def read_v1(filename):
```

Comment:
True. Note that as it is it checks if the duplicate point is present and then removes it, but I will change this to be optional.